pkg/nfd-topology-updater/nfd-topology-updater.go

```diff
@@ -138,7 +138,7 @@ func (w *nfdTopologyUpdater) Run() error {
 		select {
 		case <-crTrigger.C:
 			klog.Infof("Scanning")
-			podResources, err := resScan.Scan()
+			podResources, annotations, err := resScan.Scan()
```

Comment:
I think we should add some preparatory refactoring letting `Scan()` here return a `struct` like `ScanResponse` which will hold `[]PodResources` initially and then also `map[string]string`